virttest/utils_net.py

```diff
@@ -1609,21 +1613,24 @@ def get_vm_network_cfg_file(vm, iface_name):
     return iface_cfg_file
 
 
-def create_vm_network_script(vm, iface_name, mac_addr, boot_proto, net_mask,
-                             ip_addr=None, **dargs):
+def create_network_script(iface_name, mac_addr, boot_proto, net_mask,
+                          vm=None, ip_addr=None, **dargs):
     """
-    Form network script with its respective network param for vm instance
+    Form network script with its respective network param for vm or Host.
```

Comment:
View from function name,  this function use to generate configuration file to network service, but  checked it implementation, it looks not and it just return  param list. And the function is not complete.